### ISSUE TITLE
[MU4 Issue] Multiple Acciaccaturas play too fast

### ISSUE THREAD

**Issue by LuisLiszt** [musescore/MuseScore#14871]
**Describe the bug**
It's great that multiple Appoggiaturas play as Acciaccaturas now, however they are playing way too fast, especially when you compare it to MS3.

**To Reproduce**
1. Replace notes like in the video.

**Expected behavior**
Multiple Acciaccaturas should play slower.

**Screenshots**
MS3:

https://user-images.githubusercontent.com/55204261/205087079-41750494-d683-46d5-ae64-c87fc9e76190.mp4

MS4:

https://user-images.githubusercontent.com/55204261/205087735-20bf257d-e00f-4636-a703-4117bcbf4e10.mp4

**Platform information**
 - OS: Windows

**Comment by Tantacrul** [comment]
Yeah, I think a little refinement would be nice. In particular, I'm not hearing the last 'D' in each example in V4

**Comment by LuisLiszt** [comment]
@Tantacrul that might be related to your issue #14051 ... we'll see if the fix for this will also fix my issue here...

**Comment by DmitryArefiev** [comment]
In master (4.4) after #20453, it sounds slower comparing with MS4.3 and MS3.6.2


https://github.com/musescore/MuseScore/assets/90187801/827410b8-1165-4484-a5ec-2dcd627d9f20
